instrumentation/github.com/gorilla/mux/otelmux/config.go

```diff
@@ -56,3 +59,13 @@ func WithTracerProvider(provider oteltrace.TracerProvider) Option {
 		}
 	})
 }
+
+// WithSpanNameFormatter specifies a function to use for generating a custom span
+// name. By default, the route name (path template or regexp) is used. The route
+// name is provided so you can use it in the span name without needing to
+// duplicate the logic for extracting it from the request.
+func WithSpanNameFormatter(fn func(routeName string, r *http.Request) string) Option {
```

Comment:
I don't think you need to name the arguments here.

```suggestion
func WithSpanNameFormatter(fn func(string, *http.Request) string) Option {
```

Reply:
They're not needed, no, but IMO they provide useful context. Plus, this follows the same convention used in https://pkg.go.dev/go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp#WithSpanNameFormatter